zigpy/device.py

```diff
@@ -338,13 +338,27 @@ def __getitem__(self, key):
         return self.endpoints[key]
 
     def get_signature(self):
+        # return the device signature by providing essential device information
+        #    - Model Identifier ( Attribut 0x0005 of Basic Cluster 0x0000 )
+        #    - Manufacturer Name ( Attribut 0x0004 of Basic Cluster 0x0000 )
+        #    - Endpoint list
+        #        - Profile Id, Device Id, Cluster Out, Cluster In
         signature = {}
+        if self._manufacturer is not None:
+            signature["manufacturer_name"] = self.manufacturer
+        if self._model is not None:
+            signature["model"] = self._model
+        if self.node_desc.is_valid:
+            signature["node_descriptor"] = self._node_desc.as_dict()
```

Comment:
My bad. it is supposed to be 
```suggestion
            signature["node_descriptor"] = self.node_desc.as_dict()
```